main.go

```diff
@@ -42,6 +56,9 @@ func main() {
 			pathPrefix: *pathPrefix,
 		}
 
+		graphiteFeeder := NewGraphiteFeeder(*graphiteHost, *graphitePort, *environment, controller)
```

Comment:
Could the `GraphiteFeeder` use a URL instead of passing the port and host all over?